src/os/bluestore/BlueStore.cc

```diff
@@ -14241,14 +14261,19 @@ int BlueStore::_do_alloc_write(
 
     PExtentVector extents;
     int64_t left = final_length;
+    bool has_chunk2defer = false;
     while (left > 0) {
       ceph_assert(prealloc_left > 0);
+      has_chunk2defer |= (prealloc_pos_length < prefer_deferred_size.load());
```

Comment:
s/prefer_deferred_size.load()/prefer_deferred_size_snapshot
I did not notice that when I wrote this code.